CHANGES.rst

```diff
@@ -1751,6 +1751,8 @@ Bug Fixes
     raises a ``TypeError`` on them (while still allowing ``Time`` arrays to be
     iterated). [#4048]
 
+  - Added a new example of a custom time format class ``ydate_custom``. [#4703]
```

Comment:
Correct, pure documentation updates don't normally get a changelog entry.  So take that out.